htdocs/comm/mailing/cibles.php

```diff
@@ -332,16 +328,15 @@
 				// Si le module mailing est qualifie
 				if ($qualified)
 				{
-					$var = !$var;
 
 					if ($allowaddtarget)
 					{
-						print '<form '.$bctag[$var].' name="'.$modulename.'" action="'.$_SERVER['PHP_SELF'].'?action=add&id='.$object->id.'&module='.$modulename.'" method="POST" enctype="multipart/form-data">';
+						print '<form class="oddeven" name="'.$modulename.'" action="'.$_SERVER['PHP_SELF'].'?action=add&id='.$object->id.'&module='.$modulename.'" method="POST" enctype="multipart/form-data">';
```

Comment:
$bctag is
"impair tagtr"   or  "pair tagtr"
so we must replace with
"oddreplace tagr" and not with "oddeven"